src/to_mathlib/analysis/inner_product_space/test.lean

```diff
@@ -0,0 +1,16 @@
+import analysis.inner_product_space.basic
```

Comment:
Did you intend to keep this file? It looks redundant.

Reply:
removed